tests/client/amo/test_utils.js

```diff
@@ -0,0 +1,69 @@
+import createStore from 'amo/store';
+import * as landingActions from 'amo/actions/landing';
+import * as api from 'core/api';
+import { loadLandingAddons } from 'amo/utils';
+
+
+describe('AMO utils loadLandingAddons()', () => {
```

Comment:
mocha output isn't as nice as node/py.test but when you're hunting for a test file, using real file paths is more helpful. I'd do it like this:
````javascript
describe('amo/utils', () => {
  describe('loadLandingAddons()', () => {
    ...
  });
});
````